setup.cfg

```diff
@@ -99,7 +100,7 @@ ethereum =
 console_scripts =
     ethereum-spec-lint = ethereum_spec_tools.lint:main
     ethereum-spec-sync = ethereum_spec_tools.sync:main
-    ethereum-spec-diff = ethereum_spec_tools.diff:main
+    ethereum-spec-diff = ethereum_spec_tools.doc.diff:main
```

Comment:
Hm... so far I've tried to keep a one-to-one correspondence between a single file/folder in `src/ethereum_spec_tools` and a single installed binary, to try to keep things understandable. I think I'd like to maintain that, at least until there are too many interdependencies.

Reply:
Ok. Updated